src/renderer/i18n/en/app.json

```diff
@@ -1068,8 +1070,8 @@
     },
     "writeSeed": {
       "initialize": {
-        "title": "Save your recovery phrase",
-        "desc": "Your device will display a 24-word recovery phrase to back up your private keys",
+        "title": "Write down your recovery phrase",
+        "desc": "Your 24-word recovery phrase is the <1><0>only backup</0></1> of your private keys.",
```

Comment:
Text bold not working

**Onboarding:**

1. Set up as new device
2. Ledger Nano X
3. Continue x 2

![Capture d’écran 2020-01-23 à 17 11 05](https://user-images.githubusercontent.com/16896559/73001852-647da000-3e03-11ea-8579-8dae0b25283d.png)

Reply:
I'll have to address this on a separate pr @guycaprio good catch, I missed that